lms/djangoapps/lti_provider/views.py

```diff
@@ -86,6 +87,18 @@ def lti_launch(request, course_id, usage_id):
     params['course_key'] = course_key
     params['usage_key'] = usage_key
 
+    # Verify that the email from the LTI Launch and the logged-in user are the same.
```

Comment:
I think this logic shouldn't be handled by a view but directly in the `authenticate_lti_user`. We can raise an exception when this happens and then catch it + render the error in the view.